source/AfterburnerUsage.h

```diff
@@ -0,0 +1,47 @@
+/* AfterburnerUsage.h
+Copyright (c) 2022 by Hurleveur
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef AFTERBURNER_USAGE_H_
+#define AFTERBURNER_USAGE_H_
+
+#include "Outfit.h"
+
+
+
+// Class used to manage afterburners and their cooldowns/usages on ships.
+class AfterburnerUsage {
+public:
+	// Constructor
+	AfterburnerUsage(const Outfit &outfit);
```

Comment:
Always good to have a default constructor, just in case.

Reply:
Idk, look at AI it does not have a default constructor. This makes no sense without an attached outfit, and btw the default constructor is there if you dont delete it I think?